cli-plugins/plugin/plugin.go

```diff
@@ -24,12 +32,45 @@ import (
 // called.
 var PersistentPreRunE func(*cobra.Command, []string) error
 
+// closeOnParentSocketClose connects to the socket specified
+// by the CLI_PLUGIN_EXIT_SOCKET env var, if present, and attempts
```

Comment:
Don't forget to update this comment if we decide to change the name (see other comment)